Haskell/MaîtreDTests.hs

```diff
@@ -0,0 +1,61 @@
+{-# OPTIONS_GHC -fno-warn-orphans #-}
+
+module MaîtreDTests
+    ( tryAcceptBehavesCorrectlyWhenItCanAccept
+    , tryAcceptBehavesCorrectlyWhenItCanNotAccept
+    ) where
+
+import           Control.Monad      (liftM2)
+import           Data.Maybe         (isNothing)
+import           Data.Time          (LocalTime (..), ZonedTime (..),
+                                     defaultTimeLocale, iso8601DateFormat,
+                                     midnight, parseTimeOrError, utc)
```

Comment:
Yes they are. I eagerly applied the refactoring in https://github.com/ploeh/dependency-rejection-samples/pull/3/commits/b34c3c2bf0700dabac968bb79dd635547dbff23a and forgot to pay attention to what hsdev reports.

(ghc-mod in Atom's ide-haskell didn't report it.)